test/run

```diff
@@ -9,6 +9,11 @@ export RUN_TESTS_OPTIONS=--track-naughties
 TEST_SCENARIO=${TEST_SCENARIO:-}
 [ "${TEST_SCENARIO}" = "${TEST_SCENARIO##firefox}" ] || export TEST_BROWSER=firefox
 
+if [[ "${TEST_SCENARIO}" =~ "copr" ]]; then
```

Comment:
Ah wait, can this be anchored to the string start? Like this it's a substring match, no?

Reply:
It is a substring match, but if we make it a start string match then well /devel/copr won't work anymore

Now it is made to have that work.

Reply:
Yeah, ok -- in cockpit we split scenarios with dashes actually, i.e. "storage-devel" or so. Not a biggie, if we ever get a scenario which as "copr" as a substring, we can still fix this :wink: